.github/workflows/ci.yml

```diff
@@ -301,6 +301,11 @@ jobs:
           # refresh token before Saturday, May 25, 2024
           NPM_TOKEN: ${{ secrets.NPM_RELEASE_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.NPM_RELEASE_TOKEN }}
+      - name: Create Release Pull Request
+        # Create release notes
+        uses: changesets/action@v1
```

Comment:
createGithubReleases - A boolean value to indicate whether to create Github releases after publish or not. Default to true

https://github.com/changesets/action